File: support_tools/swigwin-3.0.12/Examples/test-suite/python/imports_runme.py
# This is the import runtime testcase.

import imports_b
import imports_a
import sys

x = imports_b.B()
imports_a.A.hello(x)

a = imports_a.A()

c = imports_b.C()
a1 = c.get_a(c)
a2 = c.get_a_type(c)

if a1.hello() != a2.hello():
    raise RuntimeError

File: support_tools/swigwin-3.0.12/Examples/test-suite/go/imports_runme.go
// This is the import runtime testcase.

package main

import "imports_b"
import "imports_a"

func main() {
	x := imports_b.NewB()
	x.Hello()

	_ = imports_a.NewA()

	c := imports_b.NewC()
	_ = c.Get_a(c)
	_ = c.Get_a_type(c)
}
